contracts/Timelock.sol

```diff
@@ -1,13 +1,13 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.15;
+pragma solidity ^0.8.20;
 
 import "@openzeppelin/contracts/governance/TimelockController.sol";
 
 /// @title Timelock - Smart contract for the timelock
 /// @author Aleksandr Kuperman - <aleksandr.kuperman@valory.xyz>
-/// @dev The OpenZeppelin functions are used as is, using the last OZ version (4.6) on the date of deployment.
+/// @dev The OpenZeppelin functions are used as is, bumped to the last OZ version (4.9.1) to date.
 contract Timelock is TimelockController {
     constructor(uint256 minDelay, address[] memory proposers, address[] memory executors)
-        TimelockController(minDelay, proposers, executors)
+        TimelockController(minDelay, proposers, executors, msg.sender)
```

Comment:
This change is required for the compatibility with the latest OZ implementation, however the Timelock contract will not be modified on-chain.